modules/core/monix/src/weaver/monixcompat/suites.scala

```diff
@@ -40,8 +39,8 @@ trait SimpleMutableTaskSuite extends MutableTaskSuite {
 }
 
 trait FunTaskSuite
-    extends BaseTaskSuite
-    with FunSuiteAux[Task]
+    extends FunSuiteAux[Task]
```

Comment:
I actually think I messed up the hierarchy - FunSuiteAux should use a type member, so that it stays an integration point for something like Discipline.

Really hope it's not going to mess up the hierarchy, so I'm going to PR it into this branch.